apps/client/src/app/[locale]/(protected)/dashboard/components/PendingOrders.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import React from 'react';
+import TrendingUpIcon from 'mdi-react/TrendingUpIcon';
+
+const PendingOrders = () => (
+  <div className="col-span-1 xl:col-span-1 lg:col-span-2 md:col-span-3"> 
+    <div className="bg-white pt-[25px] pb-[38px] px-[25px] rounded-lg md:h-[125px]"
+    style={{ boxShadow: '0 10px 30px 1px rgba(0, 0, 0, 0.06)' }}>
+      <div className="flex justify-between items-center mb-1">
+        <h2 
+          className="text-2xl font-medium font-sans"
+          style={{ color: '#4CE1B6' }}
+        >
+          25
+        </h2>
+        <TrendingUpIcon className="text-gray-300 w-6 h-6" />
+      </div>
+      <h5
+        className="text-[12px] font-medium text-left opacity-70 mt-[-4px] uppercase text-primary-600 font-sans"
```

Comment:
Check what we have in tw-config when using font.

Reply:
> Check what we have in tw-config when using font.